### PR TITLE
Feature/15 table alignment

### LINKED ISSUE #15: Transforms to set alignment for a column
This plugin should expose a `transforms.setColumnAlign(at: Number, align: String)`

### PR DESCRIPTION
**Pull request by mklabs**
Should fix #15 

This PR adds a new transform to set the column alignment for a given column, which gets added as a data property in the schema in the form of 

```
data: {
  align: 'left' | 'center' | 'right'
}
```

### CODE REVIEW

**Comment by SamyPesse** [comment]
The plugin should export the alignment values as constants (`ALIGN.RIGHT`, etc).

**Comment by mklabs** [comment]
@SamyPesse true, will do 👍

File: README.md [hunk 1]
```diff
@@ -91,3 +91,10 @@ Move the selection to a specific position in the table.
 `plugin.transforms.moveSelectionBy(transform: Transform, column: Number, row: Number) => Transform`
 
 Move the selection by the given amount of columns and rows.
+
+#### `transforms.setColumnAlign`
+
+`plugin.transforms.setColumnAlign(transform: Transform, align: String, at: Number) => Transform`
+
+Sets column alignment for a given column (`at`). `align` defaults to center,
```
Comment (Soreine) [inline README.md]:
```
...for a given column (`at`), in the current table.
```

File: example/main.js [hunk 2]
```diff
@@ -87,6 +90,15 @@ const Example = React.createClass({
         );
     },
 
+    onSetAlign: function (event, align) {
+        let { state } = this.state;
+
+        this.onChange(
+            tablePlugin.transforms.setColumnAlign(state.transform(), null, align)
```
Comment (Soreine) [inline example/main.js]: Update order of arguments

File: lib/transforms/setColumnAlign.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+function setColumnAlign(opts, transform, align = 'center', at) {
+    const { state } = transform;
+    const { startBlock } = state;
+
+    const pos = TablePosition.create(state, startBlock);
+    const { table } = pos;
+
+    // Figure out column position, defaults 0 position to 1, any other falsy
+    // values defaults to current column index
+    if (at === 0) {
```
Comment (Soreine) [inline lib/transforms/setColumnAlign.js]: `at` should be 0 based too

**Pull request merged**